tools/doc/html.js

```diff
@@ -165,10 +175,32 @@ function parseLists(input) {
 }
 
 
+// Syscalls which appear in the docs, but which only exist in BSD / OSX
+var BSD_ONLY_SYSCALLS = ['lchmod'];
+
+// Handle references to man pages, eg "open(2)" or "lchmod(2)"
+// Returns modified text, with such refs replace with HTML links, for example
+// '<a href="http://man7.org/linux/man-pages/man2/open.2.html">open(2)</a>'
+function linkManPages(text) {
+  return text.replace(/ ([a-z]+)\((\d)\)/gm, function(match, name, number) {
+    // name consists of lowercase letters, number is a single digit
+    var displayAs = name + '(' + number + ')';
+    if (BSD_ONLY_SYSCALLS.indexOf(name) > -1) {
+      return ' <a href="http://www.freebsd.org/cgi/man.cgi?query=' + name +
+        '&sektion=' + number + '">' + displayAs + '</a>';
+    } else {
+      return ' <a href="http://man7.org/linux/man-pages/man' + number + '/' +
+        name + '.' + number + '.html">' + displayAs + '</a>';
```

Comment:
I was curious about that as well. I went through some digging on Man pages, so that particular set of docs is linked to from [kernel.org](https://www.kernel.org/doc/man-pages/) as well. There's a bunch more sources for the Linux docs but I can't really find any official source. [die.net](http://linux.die.net/man/) for example.